apps/openmw/mwclass/light.cpp

```diff
@@ -159,11 +159,9 @@ namespace MWClass
 
         if (Settings::Manager::getBool("show effect duration","Game"))
             text += "\n#{sDuration}: " + MWGui::ToolTips::toString(ptr.getClass().getRemainingUsageTime(ptr));
-        if (ref->mBase->mData.mWeight != 0)
-        {
-            text += "\n#{sWeight}: " + MWGui::ToolTips::toString(ref->mBase->mData.mWeight);
-            text += MWGui::ToolTips::getValueString(ref->mBase->mData.mValue, "#{sValue}");
-        }
+
+        text += MWGui::ToolTips::getValueString(ref->mBase->mData.mWeight, "#{sWeight}");
```

Comment:
I think you meant to use _getWeightString_ here (despite being equal to _getValueString_ in terms of functionality).